src/main/java/com/cleanroommc/modularui/theme/Theme.java

```diff
@@ -102,4 +120,28 @@ public WidgetTheme getWidgetTheme(String id) {
         }
         return getFallback();
     }
+
+    @Override
+    public int getOpenCloseAnimationOverride() {
+        if (this.openCloseAnimationOverride != -1) {
+            return this.openCloseAnimationOverride;
+        }
+        return ModularUIConfig.panelOpenCloseAnimationTime;
+    }
+
+    @Override
+    public boolean getSmoothProgressBarOverride() {
+        if (this.smoothProgressBarOverride != null) {
+            return this.smoothProgressBarOverride;
+        }
+        return ModularUIConfig.smoothProgressBar;
+    }
+
+    @Override
+    public Tooltip.Pos getTooltipPosOverride() {
+        if (this.tooltipPosOverride != null) {
+            return this.tooltipPosOverride;
+        }
+        return ModularUIConfig.tooltipPos;
+    }
```

Comment:
I feel like this should be inverted, so use -1 and null in the config and if the config is not -1 or null use the theme value.
This way each theme can have a default value, but if users want to they can override it for every mui screen.